security/pkg/server/ca/server.go

```diff
@@ -107,7 +107,8 @@ func (s *Server) CreateCertificate(ctx context.Context, request *pb.IstioCertifi
 		sans = []string{impersonatedIdentity}
 	}
 	certSigner := crMetadata[security.CertSigner].GetStringValue()
-	serverCaLog.Debugf("cert signer from workload %s", certSigner)
+	serverCaLog.Infof("generating a certificate for %v, signer: %s, ttl: %d", sans, certSigner,
```

Comment:
ValidityDuration is not the TTL, it is the _requested_ TTL. Big difference - users may see `ttl: 100000d` and get worried.

Reply:
Requested TTL equals the TTL in the generated cert if it's greater than 0 and less than max, otherwise a default value is used or an error is thrown out. 

https://github.com/istio/istio/blob/9d86e62a531bc603755bc210a59b9143be11986e/security/pkg/pki/ca/ca.go#L429